jobs/go/http/README.md

```diff
@@ -0,0 +1,152 @@
+
+- `job-scheduler.go`, responsible for scheduling, retrieving and deleting jobs.
+- `job-service.go`, responsible for handling the triggered jobs.
+
+## Run the app with the template file
+
+This section shows how to run both applications at once using [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) with `dapr run -f .`.  This enables to you test the interactions between multiple applications and will `schedule`, `run`, `get`, and `delete` jobs within a single process.
+
+Open a new terminal window and run the multi app run template:
+
+<!-- STEP
+name: Run multi app run template
+expected_stdout_lines:
+  - '== APP - job-service == Received job request...'
+  - '== APP - job-service == Executing maintenance job: Oil Change'
+  - '== APP - job-scheduler == Job Scheduled: C-3PO'
+  - '== APP - job-service == Received job request...'
+  - '== APP - job-service == Executing maintenance job: Limb Calibration'
+expected_stderr_lines:
+output_match_mode: substring
+match_order: none
+background: true
+sleep: 60
+timeout_seconds: 120
+-->
+
+```bash
+dapr run -f .
+```
+
+The terminal console output should look similar to this, where:
+
+- The `R2-D2` job is being scheduled.
+- The `R2-D2` job is being executed after 2 seconds.
+- The `C-3PO` job is being scheduled.
+- The `C-3PO` job is being retrieved.
+
+```text
+== APP - job-scheduler == Job Scheduled: R2-D2
+== APP - job-service == Received job request...
+== APP - job-service == Starting droid: R2-D2
+== APP - job-service == Executing maintenance job: Oil Change
+== APP - job-scheduler == Job Scheduled: C-3PO
+== APP - job-scheduler == Job details: {"name":"C-3PO", "dueTime":"30s", "data":{"@type":"ttype.googleapis.com/google.protobuf.StringValue", "expression":"C-3PO:Limb Calibration"}}
+```
+
+After 30 seconds, the terminal output should present the `C-3PO` job being processed:
+
+```text
+== APP - job-service == Received job request...
+== APP - job-service == Starting droid: C-3PO
+== APP - job-service == Executing maintenance job: Limb Calibration
+```
+
+2. Stop and clean up application processes
+
+```bash
+dapr stop -f .
+```
+
+<!-- END_STEP -->
+
+## Run the Jobs APIs individually
+
+### Schedule Jobs
+
+1. Open a terminal and run the `job-service` app:
+
+```bash
+dapr run --app-id job-service --app-port 6200 --dapr-http-port 6280 -- go run .
+```
+
+2. On a new terminal window, schedule the `R2-D2` Job using the Jobs API.
+
+```bash
+curl -X POST \
+  http://localhost:6280/v1.0-alpha1/jobs/jobforjabba \
+  -H "Content-Type: application/json" \
+  -d '{
+        "data": {
+          "@type": "type.googleapis.com/google.protobuf.StringValue",
+          "value": "R2-D2:Oil Change"
+        },
+        "dueTime": "2s"
+    }'
+    ```
+
+Back at the `job-service` app terminal window, the output should be:
+
+```text
+== APP - job-app == Received job request...
+== APP - job-app == Starting droid: R2-D2
```

Comment:
Isn't this the name of the job that is started?
```suggestion
== APP - job-app == Starting droid: R2-D2
```

Reply:
you are right, this was a bad copy/paste